src/main/resources/schema.json

```diff
@@ -180,5 +180,102 @@
                 ]
             }
         ]
+    },
+    {
+        "endpointId":"Sleep",
+        "url":"https://api.fitbit.com/1.2/user/%s/sleep/date/%s.json",
+        "urlParameters":["USER_ID", "DATE"],
+        "tables":[
+            {
+                "tableKey":"sleep",
+                "columns":[
+                    {
+                        "columnId":"dateOfSleep",
+                        "columnType":"STRING",
+                        "maxLength":10
+                    },
+                    {
+                        "columnId":"duration",
+                        "columnType":"INTEGER"
+                    },
+                    {
+                        "columnId":"efficiency",
+                        "columnType":"INTEGER"
+                    },
+                    {
+                        "columnId":"infoCode",
+                        "columnType":"INTEGER"
+                    },
+                    {
+                        "columnId":"isMainSleep",
+                        "columnType":"BOOLEAN"
+                    },
+                    {
+                        "columnId":"levels",
+                        "columnType":"LARGETEXT"
```

Comment:
Does LARGETEXT in this case capture all of the nested structures within levels?  For example, would this column be populated with the following:
`{
                "summary": {
                    "deep": {
                        "count": <value>,
                        "minutes": <value>,
                        "thirtyDayAvgMinutes": <value>
                    },
                    "light": {
                        "count": <value>,
                        "minutes": <value>,
                        "thirtyDayAvgMinutes": <value>
                    },
                    "rem": {
                        "count": <value>,
                        "minutes": <value>,
                        "thirtyDayAvgMinutes": <value>
                    },
                    "wake": {
                        "count": <value>,
                        "minutes": <value>,
                        "thirtyDayAvgMinutes": <value>
                    }
                }`

Reply:
Yes, a LARGETEXT would just copy the entire JSON blob into the Synapse table column.